packages/flutter_tools/test/general.shard/macos/cocoapods_test.dart

```diff
@@ -273,7 +273,7 @@ void main() {
         ..writeAsStringSync('Existing release config');
 
       final FlutterProject project = FlutterProject.fromPath('project');
-      await injectPlugins(project);
+      await injectPlugins(project, checkProjects: true);
```

Comment:
is this related to this PR?

Reply:
Yep. When this flag is true, `injectPlugins` skips generating the Android plugin registrant since there isn't an `android/` directory in the project under test.